src/fu-util.c

```diff
@@ -1658,6 +1941,24 @@ main (int argc, char *argv[])
 		     /* TRANSLATORS: command description */
 		     _("Install prepared updates now"),
 		     fu_util_install_prepared);
+	fu_util_add (priv->cmd_array,
+		     "history",
```

Comment:
In keeping with the other "get-" commands ("get-devices", "get-results", "get-updates", "get-details"), I think this should be "get-history".

Reply:
Fixed